google/cloud/storage/internal/grpc_object_access_control_parser.h

```diff
@@ -26,10 +26,14 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 google::storage::v2::ObjectAccessControl ToProto(
     storage::ObjectAccessControl const& acl);
+
```

Comment:
nit: the line spacing in this file is weird. I would suggest adding newlines between each function.

Reply:
Done